packages/eslint-plugin/tests/rules/no-unnecessary-type-arguments.test.ts

```diff
@@ -1,16 +1,15 @@
-import path from 'path';
 import rule from '../../src/rules/no-unnecessary-type-arguments';
-import { RuleTester } from '../RuleTester';
+import { RuleTester, getFixturesRootDir } from '../RuleTester';
+
+const rootPath = getFixturesRootDir();
 
-const rootDir = path.join(process.cwd(), 'tests/fixtures');
 const ruleTester = new RuleTester({
+  parser: '@typescript-eslint/parser',
   parserOptions: {
-    ecmaVersion: 2015,
```

Comment:
eslint has internal validation for some ast and this is requred as we are using `import` within tests

es2015 = es6

> ImportDeclaration should appear when the mode is ES6 and in the module context.

Reply:
it's just the `sourceType: 'module'` that's required.
the ecmaVersion is used by espree.